src/readfish/plugins/dorado.py

```diff
@@ -66,6 +67,18 @@ def __init__(
         self.supported_barcode_kits = None
         self.supported_basecall_models = None
         self.run_information = run_information
+        if self.run_information:
+            self.guppy_version = self.run_information.software_versions.guppy_connected_version
+
+            if parse_version(self.guppy_version) >= parse_version("7.3.9"):
+                logging.info(
+                    f"Connected to caller version {self.guppy_version}."
+                )
+            else:
+                raise RuntimeError(
+                    f"Connected to caller version {self.guppy_version}. This plugin requires a version of Dorado >= 7.3.9."
```

Comment:
Failed to connect, rather than connected?

Reply:
Changed to "Trying to connect". Strictly connection hasn't failed, but rather we can connect but sending reads will fail.